packages/document-model-libs/editors/document-model-2/document-model-editor.tsx

```diff
@@ -16,6 +16,7 @@ type Props = {
   localStateInitialValue: string;
   handlers: DocumentActionHandlers;
   modules: Module[];
+  errors: string;
```

Comment:
functionality of the component is the same, but layout has been adjusted. also shows the errors under the editor